bedrock/mozorg/tests/test_views.py

```diff
@@ -426,49 +425,38 @@ def test_with_autoresponse(self):
 
     @patch.object(ReCaptchaField, 'clean', Mock())
     @patch('bedrock.mozorg.email_contribute.basket.subscribe')
-    @patch('bedrock.mozorg.email_contribute.requests.post')
-    def test_webmaker_mentor_signup(self, mock_post, mock_subscribe):
+    def test_webmaker_mentor_signup(self, mock_subscribe):
         """Test Webmaker Mentor signup form for education functional area"""
         self.data.update(interest='education', newsletter=True)
         self.client.post(self.url_en, self.data)
 
-        assert_false(mock_subscribe.called)
-        payload = {'email': self.contact, 'custom-1788': '1'}
-        mock_post.assert_called_with('https://sendto.mozilla.org/page/s/mentor-signup',
-                                     data=payload, timeout=2)
+        mock_subscribe.assert_called_with(self.contact, 'mozilla-learning-network', source_url=ANY)
 
     @patch.object(ReCaptchaField, 'clean', Mock())
     @patch('bedrock.mozorg.email_contribute.basket.subscribe')
-    @patch('bedrock.mozorg.email_contribute.requests.post')
-    def test_webmaker_mentor_signup_newsletter_fail(self, mock_post, mock_subscribe):
+    def test_webmaker_mentor_signup_newsletter_fail(self, mock_subscribe):
         """Test Webmaker Mentor signup form when newsletter is not selected"""
         self.data.update(interest='education', newsletter=False)
         self.client.post(self.url_en, self.data)
 
         assert_false(mock_subscribe.called)
-        assert_false(mock_post.called)
 
     @patch.object(ReCaptchaField, 'clean', Mock())
     @patch('bedrock.mozorg.email_contribute.basket.subscribe')
-    @patch('bedrock.mozorg.email_contribute.requests.post')
-    def test_webmaker_mentor_signup_functional_area_fail(self, mock_post, mock_subscribe):
+    def test_webmaker_mentor_signup_functional_area_fail(self, mock_subscribe):
         """Test Webmaker Mentor signup form when functional area is not education"""
         self.data.update(interest='coding', newsletter=True)
         self.client.post(self.url_en, self.data)
 
         mock_subscribe.assert_called_with(self.contact, 'about-mozilla', source_url=ANY)
-        assert_false(mock_post.called)
 
     @patch.object(ReCaptchaField, 'clean', Mock())
     @patch('bedrock.mozorg.email_contribute.basket.subscribe')
-    @patch('bedrock.mozorg.email_contribute.requests.post')
-    def test_webmaker_mentor_signup_timeout_fail(self, mock_post, mock_subscribe):
+    def test_webmaker_mentor_signup_timeout_fail(self, mock_subscribe):
         """Test Webmaker Mentor signup form when request times out"""
-        mock_post.side_effect = Timeout('Timeout')
```

Comment:
Should this test also set a `side_effect` on `mock_subscribe` (as you did below)? It's not really testing any failures/timeouts as-is, is it?